src/azure-cli/azure/cli/command_modules/appconfig/keyvalue.py

```diff
@@ -182,18 +184,24 @@ def import_config(cmd,
     # append all feature flags to src_kvs list
     src_kvs.extend(src_features)
 
-    # In strict mode, delete kvs with specific label that are missing from the imported file
-    if strict:
+    if strict or import_mode == ImportMode.IGNORE_MATCH:
         dest_kvs.extend(all_features)  # append the discarded features back
 
+    # In strict mode, delete kvs with specific label that are missing from the imported file
+    if strict:
         kvs_to_delete = list(filterfalse(lambda kv: any(kv_import.key == kv.key and label == kv.label
                                                         for kv_import in src_kvs), dest_kvs))
         for kv in kvs_to_delete:
             __delete_configuration_setting_from_config_store(azconfig_client, kv)
 
     # import into configstore
+    kvs_to_write = copy(src_kvs)
```

Comment:
Why do we need shallow copy here?

Reply:
This was initially added since it appears the key_values array can be mutated within the `__write_kv_and_features_to_config_store` method. But since neither kvs_to_write nor dest_kvs is used after this call, there should be no need for shallow copying.